pandas/core/arrays/datetimelike.py

```diff
@@ -2151,6 +2174,43 @@ def unit(self) -> str:
         return dtype_to_unit(self.dtype)  # type: ignore[arg-type]
 
     def as_unit(self, unit: str, round_ok: bool = True) -> Self:
+        """
+        Convert to a dtype with the given unit resolution.
+
+        Parameters
+        ----------
+        unit : {'s', 'ms', 'us', 'ns'}
+        round_ok : bool, default True
+            If False and the conversion requires rounding, raise.
```

Comment:
```suggestion
            If False and the conversion requires rounding, raise ValueError.
```

Reply:
Thanks for catching this! committed a fix.